packages/kbn-unified-data-table/src/components/column_header_truncate_container.tsx

```diff
@@ -15,7 +15,7 @@ const ColumnHeaderTruncateContainer = ({
   headerRowHeight = 1,
   children,
 }: {
-  headerRowHeight: number;
+  headerRowHeight?: number;
```

Comment:
This function has the default value logic, which means the type can be made optional.